server/admin/src/main.rs

```diff
@@ -48,7 +48,9 @@ pub enum FeatureFlag {
 async fn main() {
     let config = Config::from_env_vars();
     let (index_db_pool, files_db_client) = connect_to_state(&config).await;
-    let server_state = ServerState { config, index_db_pool, files_db_client };
+    let stripe_client = stripe::Client::new("");
```

Comment:
I think we will need to do real things with strip in admin, and we already have a real config object, just pipe it through like you'd expect.